src/components/RecoveryPhraseInput.tsx

```diff
@@ -68,8 +69,8 @@ export default function RecoveryPhraseInput({
   return (
     <Card
       rounded={true}
-      shadow={showInput ? Shadow.SoftLight : null}
-      style={[showInput ? styles.containerActive : styles.container]}
+      shadow={!isValidBackupPhrase(inputValue) ? null : Shadow.SoftLight}
+      style={[!isValidBackupPhrase(inputValue) ? styles.containerActive : styles.container]}
```

Comment:
can we confirm with design if this was actually intended? I think its cleaner to just have different styles when input is possible or not.

Reply:
confirmed with design that it can just change when input is possible or not, changed it in [19f18d2](https://github.com/valora-inc/wallet/pull/5628/commits/19f18d27432853ad5f95bc3cc7ef1f9f6fa5eea7)